typescript-api/scripts/postbuild.ts

```diff
@@ -0,0 +1,33 @@
+import { writeFileSync, copyFileSync, readFileSync } from "fs";
+import path from "path";
+
+async function main() {
+  // console.log("Loading package.json");
+
+  // const pck = JSON.parse(readFileSync(path.join(process.cwd(), "package.json"), "utf-8"));
+  const buildPath = `${process.env.PWD}/build`;
+
+  // pck.scripts = {};
+  // pck.private = false;
+  // pck.type = "module";
+  // pck.files = ["**/*", "!**/tsconfig.tsbuildinfo", "!**/*.tgz"];
+
+  // console.log(`Writing ${buildPath}/package.json`);
+  // writeFileSync(`${buildPath}/package.json`, JSON.stringify(pck, null, 2));
```

Comment:
Are there commentefd for some reason?

Reply:
Yes, because before this script was trying to handcraft a package.json via find and replace. Instead we should just use a package manager like god would intend.